README.md

```diff
@@ -1 +1,249 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+ A New developer can view my project updates on Git HUB shared Repository
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+> Browser
+> GitHub Account
+> VS Code Editor
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder: 
+git clone <project repository link>
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+
+
+### Deployment
+
+ This project can be deployed using:
+Git Hub 
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project. (No Collaborators - Solo Capstone Project)
+Author - Baqar Abbas - Author1: Baqar Abbas 
+<ul>
+    <li>GitHub<a href="https://github.com/">@githubhandle</a></li>
+    <li>Twitter<a href="https://www.twitter.com/">@twitterhandle</a></li>
+    <li>LinkedIn<a href="https://www.linkedin.com"">@LinkedIn</a></li>
+  </ul>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features will be added and enhancements will be updated<a name="future-features"></a>
+
+> Desktop and Mobile responsive Design of Conference Site project..
+Conference Site project will be updated to handle responsiveness so that it will look good on all major breakpoints
+of resolution for handling Desktop and mobile resolution. It will be handled using media queries.
+
+- [ ] **[Conference Site Project]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+ This is a Conference Site Project following the Git Flow Processes.
+If you like this project give it a Thumbs up ...
+This Project shows Conference page of Web Summit and build on a responsive design.
+using mobile first approach _ with responsive desgins for small and large screens resolutions.
+Live Demo will be available on GitHub.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank the Code Reveiwer as well for their constructive feedback...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ OPTIONAL] Your README should contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
All the optional changes I will incorporate them later on. Please consider the time factor I also have coding challenges to complete for this week. This is my fourth review that I am sending again for a review please approve